### PR TITLE
23514 Override input label and text + fix pronoun

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#23514

*Description of changes:*
- app version = 5.12.4
- imported new shared Approval Type component (with lint fix)
- imported new shared Certify component (with pronoun and colour fix)
- updated entity display (per Jacqueline)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-746-3mpo8a83.web.app

File: package.json [hunk 1]
```diff
@@ -14,11 +14,11 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.21.5",
-    "@bcrs-shared-components/approval-type": "1.1.2",
+    "@bcrs-shared-components/approval-type": "1.1.3",
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.15",
     "@bcrs-shared-components/business-lookup": "1.3.4",
-    "@bcrs-shared-components/certify": "2.1.15",
+    "@bcrs-shared-components/certify": "2.1.51",
```
Comment (severinbeauvais) [inline package.json:21]: The new shared component changed "is authorized" to "are authorized" and fixed the input colours per accessibility standards.

Before:
image

After:
image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-746-3mpo8a83.web.app

File: src/resources/ContinuationIn/C.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ export const ContinuationInResourceC: ContinuationInResourceIF = {
     completingPartyStatement: {
       certifyStatements: [],
       certifyClause: ResourcePhrases.OFFENCE_SECTION_427,
-      entityDisplay: null
+      entityDisplay: 'business'
```
Comment (severinbeauvais) [inline src/resources/ContinuationIn/C.ts:38]: Before:
image

After:
image

File: package.json [hunk 3]
```diff
@@ -14,11 +14,11 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.21.5",
-    "@bcrs-shared-components/approval-type": "1.1.2",
+    "@bcrs-shared-components/approval-type": "1.1.3",
```
Comment (severinbeauvais) [inline package.json:17]: Just a lint fix in this shared component.

**Review by ArwenQin — APPROVED** [review]
Looks good

**Pull request merged**